### PR TITLE
light: 1.1.2 -> 1.2, use new udev support instead of setuid wrapper.

### PR DESCRIPTION
**Pull request by dtzWill**
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/modules/programs/light.nix [hunk 1]
```diff
@@ -13,14 +13,14 @@ in
...
+          Whether to install Light backlight control command and udev rules.
         '';
       };
     };
   };
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ pkgs.light ];
-    security.wrappers.light.source = "${pkgs.light.out}/bin/light";
+    services.udev.packages = [ pkgs.light ];
```
Comment (Mic92) [inline nixos/modules/programs/light.nix]: Do I need to be in a specific group for that?

Comment (Mic92) [reply nixos/modules/programs/light.nix]: Ok. I added this to our release notes.

File: nixos/doc/manual/release-notes/rl-1903.xml [hunk 2]
```diff
@@ -105,6 +105,14 @@
      <varname>rabbitmq-server</varname>.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       The <literal>light</literal> module no longer use setuids binary, but
```
Comment (Mic92) [inline nixos/doc/manual/release-notes/rl-1903.xml]: typo I introduced: setuid binaries

**Comment by dtzWill** [comment]
This is good to go, I think? Just checking :).

**Comment by Mic92** [comment]
I have not really tested it myself by adding myself to group, but I suppose you did, so should be good to merge then.

**Comment by dtzWill** [comment]
Yep use it every day :).

**Pull request merged**